src/store/workflows.module.js

```diff
@@ -40,6 +40,28 @@ function _getCycles (workflows) {
   return cycles
 }
 
+/**
+ * Compute percent progress.
+ *
+ * @see https://github.com/cylc/cylc-flow/blob/de7d938496e82dbdfb165938145670dd8e801efd/lib/cylc/gui/updater_tree.py#L248-L263
+ * @param {number} startedTime in milliseconds since 1970-01-01 00:00:00 UTC, e.g. 1568353099874
+ * @param {number} meanElapsedTime mean elapsed time (e.g. 13.5)
+ * @returns {number} the percent progress, e.g. 25 (meaning 25% progress)
+ * @private
+ */
+function _computePercentProgress (startedTime, meanElapsedTime) {
+  const now = Date.now()
+  // startedTime > now reportably possible via interaction with `cylc reset`
+  if (startedTime > now || meanElapsedTime === 0) {
+    return 0
+  }
+  const elapsedTime = startedTime + meanElapsedTime
```

Comment:
This variable is "expected time of completion" so "elapsedTime" isn't a good name for it. ("Elapsed time" is how much time has passed so far, while the job is running).

Reply:
Aaaahh! I was trying to guess what was `tetc_unix `! Will change it, thanks!

Reply:
I had to remind myself.  Bad "etc" was a bad variable name in the old code, sorry!